src/databricks/sql/thrift_backend.py

```diff
@@ -152,8 +152,12 @@ def __init__(
             ssl_context=ssl_context,
         )
 
-        timeout = kwargs.get("_socket_timeout")
-        # setTimeout defaults to None (i.e. no timeout), and is expected in ms
+        timeout = (
+            60
+            if kwargs.get("_socket_timeout") is None
+            else kwargs.get("_socket_timeout")
+        )
+        # setTimeout defaults to 60 seconds and is expected in ms
```

Comment:
The `dict.get()` method in Python returns `None` if the key isn't present and accepts a default value as its second argument. So this can be simplified like so:

```suggestion
        DEFAULT_SOCKET_TIMEOUT = float(60)
        timeout = kwargs.get("_socket_timeout",  DEFAULT_SOCKET_TIMEOUT)

        # setTimeout defaults to 60 seconds and is expected in ms
```